spec/models/article_spec.rb

```diff
@@ -137,3 +137,28 @@
     end
   end
 end
+
```

Comment:
Not random. I just want to look at the past 30 days vs the 30 days prior. I want to be able to  judge roughly which way we are trending.  If I instead attempt to compare February to March, the comparison is not meaningful because of the 3-day discrepancy.  If we were talking about financial accounting or something that required reporting on specific months, it would be different, but as it is, I don't think it makes sense to do what you are talking about.   I chose 30 vs 30 over idiosyncratic month v month in this context for the simple reason that we need to make a quick, meaningful comparison to determine which way we are going.